src/styles/colors.js

```diff
@@ -1,13 +1,13 @@
 const colors = {
-  colorPrimary: '#3f51b5',
+  colorPrimary: '#A6F3D6',
   colorPrimaryLight: '#757de8',
-  colorPrimaryDark: '#002984',
+  colorPrimaryDark: '#07377A',
   colorSecondary: '#f2f4ff',
```

Comment:
Não estamos usando essa `colorSecondary`. Substituir pelo valor da `colorPrimaryDark` e remover `colorPrimaryDark`.

Remover também as cores;  `colorSecondaryLight: '#ffffff',` e   `colorSecondaryDark: '#bfc1cc',`

Não é legal usar cores com nomes light e dark, pois não estamos usando dois temas e sim apenas um.

```suggestion
  colorSecondary: '#07377A',
```